app/static/tests/mocks.ts

```diff
@@ -75,6 +76,20 @@ export const mockFitDataState = (state:Partial<FitDataState> = {}): FitDataState
     };
 };
 
+export const mockSensitivityState = (state: Partial<SensitivityState> = {}): SensitivityState => {
+    return {
+        paramSettings: {
+            parameterToVary: null,
```

Comment:
this is picking up the first parameter by default - that's fine but eventually do you think this should be empty to start? (probably not worth doing in this PR)

Reply:
We should probably default to something for all values or for none. Feels a bit lower friction to default to something that will work right away?